src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -116,21 +119,28 @@ class PropertyAccessor implements PropertyAccessorInterface
      * @var array
      */
     private $writePropertyCache = array();
+    
     private static $previousErrorHandler = false;
     private static $errorHandler = array(__CLASS__, 'handleError');
     private static $resultProto = array(self::VALUE => null);
 
     /**
+     * @var ClassMetadataFactoryInterface
+     */
```

Comment:
I'd remove it like I've done in other places following your remarks, but #16838 add other definitions the same way in the same class. It would be awkward to have some and not others. Should I leave it or remove all?